framework/wazuh/core/batcher/client.py

```diff
@@ -0,0 +1,57 @@
+import uuid
+import asyncio
+from typing import Optional
+
+from wazuh.core.batcher.mux_demux import MuxDemuxQueue
+
+
+class BatcherClient:
+    """
+    A client for sending and receiving events via a MuxDemuxQueue.
+
+    Parameters
+    ----------
+    queue : MuxDemuxQueue
+        The MuxDemuxQueue instance used to route messages.
+    """
+    def __init__(self, queue: MuxDemuxQueue, frequency_of_wait: int = 0.1):
+        self.queue = queue
+        self.frequency_of_wait = frequency_of_wait
+
+    def send_event(self, event) -> uuid.UUID:
+        """
+        Sends an event through the RouterQueue.
+
+        Parameters
+        ----------
+        event : any
+            The event to be sent.
+
+        Returns
+        -------
+        uuid.UUID
+            The unique identifier assigned to the event.
+        """
+        assigned_uid = uuid.uuid4()
+        return self.queue.send_to_mux(assigned_uid, event)
```

Comment:
I think we can use an ID that is less expensive than UUID, the amount of simultaneous events in a batcher is limited and they are consuming unnecessary memory. We could stick to using the `id()` built-in [function](https://docs.python.org/3/library/functions.html#id) which guarantees uniqueness on objects whose lifetime is overlapping

```suggestion
        assigned_uid = uuid.uuid4()
        return self.queue.send_to_mux(assigned_uid, event)
```

Reply:
Changed in 92c3106a91132f0a82fbc5b27062532886153f28.